dapps/web3modal/next/biome.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.5.2/schema.json",
+	"organizeImports": {
+		"enabled": false
+	},
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": true
+		}
+	},
+	"javascript": {
+    "formatter": {
+      "semicolons": "asNeeded",
+			"quoteStyle":"single",
+			"lineWidth": 120
+    }
+	}
+}
```

Comment:
what's with this lint?

Reply:
very opinionated of mine, should I remove it? :v